protected/humhub/modules/space/modules/manage/views/security/index.php

```diff
@@ -24,7 +24,7 @@
 
         <?php
         $visibilities = [
-            Space::VISIBILITY_NONE => Yii::t('SpaceModule.base', 'Private (Invisible)'),
+            Space::VISIBILITY_NONE => Yii::t('SpaceModule.base', 'Private (Invisible, All Contents would be private)'),
```

Comment:
I think `All content becomes private` is better.

Reply:
We should remove that and put the notice in the Confirm dialogue.